src/otx/algo/callbacks/tile_sampling_hook.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Tile Sampling Hook for OTXTileTrainDataset."""
+from lightning import Callback, LightningModule, Trainer
+
+
+class TileSamplingHook(Callback):
```

Comment:
Personally, I prefer to use the word "Callback" for the OTX2.0
```suggestion
class TileSamplingCallback(Callback):
```